values.yaml

```diff
@@ -539,38 +544,48 @@ postgresql:
 # KEYCLOAK
 # ----------------------------------------------------------------------
 keycloak:
-  user: keycloak
-  password: keycloak
-  postgresql:
-    enabled: false
+  httpRelativePath: /auth/
+  auth:
+    adminUser: admin
+    adminPassword: admin
+  proxy: edge
   ingress:
-    tls: []
     enabled: true
+    tls: []
     servicePort: http
-    hosts:
-      - incore.example.com
+    hostname: incore.example.com
     rules:
       - host: incore.example.com
-        paths:
-          - path: /auth/
-            pathType: ImplementationSpecific
-  extraEnv: |
-    - name: PROXY_ADDRESS_FORWARDING
-      value: "true"
-    - name: KEYCLOAK_USER
-      value: keycloak
-    - name: KEYCLOAK_PASSWORD
-      value: keycloak
-    - name: DB_VENDOR
-      value: postgres
-    - name: DB_ADDR
-      value: incore-postgresql
-    - name: DB_PORT
-      value: "5432"
-    - name: DB_DATABASE
-      value: keycloak
-    - name: DB_USER
-      value: keycloak
-    - name: DB_PASSWORD
-      value: keycloak
-
+        http:
+          paths:
+            - path: /auth/
+              pathType: ImplementationSpecific
+  service:
+    type: ClusterIP
+  postgresql:
+    enabled: false
+  externalDatabase:
+    host: incore-postgresql
```

Comment:
can you add a comment here that if the chart name is changed this needs to change as well.

Reply:
added comment